sample_data_config/web-api/web-api-data-pipeline.config.yaml

```diff
@@ -20,16 +20,63 @@ webApi:
         - items
   #elifescience profile api
   - dataset: '{ENV}'
-    table: atest_profile_api
+    table: test_people_api
```

Comment:
was this change intentional? now there seem to be two people api configuration items

Reply:
non-intenational

Reply:
Maybe revert this back to profile?